datacenter/ucp/2.1/guides/images/architecture-2.svg

```diff
@@ -22,8 +22,8 @@
                         </g>
                         <g id="engine" transform="translate(1.000000, 79.000000)">
                             <rect id="Rectangle-138" fill="#1488C6" x="0" y="0" width="95" height="22" rx="2"></rect>
-                            <text id="CS-Docker-Engine" font-family="OpenSans, Open Sans" font-size="10" font-weight="normal" fill="#FFFFFF">
-                                <tspan x="7.26025391" y="15">CS Docker Engine</tspan>
+                            <text id="Docker-Enterprise-Engine" font-family="OpenSans, Open Sans" font-size="8" font-weight="normal" fill="#FFFFFF">
+                                <tspan x="5.26025391" y="15">Docker Enterprise Edition</tspan>
```

Comment:
Can you change this from "Docker Enterprise Edition" to "Docker EE"? Right now the text is too long and overflowing.